src/ain/node/btc-client.ts

```diff
@@ -7,25 +7,21 @@ export class BtcClient extends NodeClient {
     super(http, url, scheduler, mode);
   }
 
-  async send(addressTo: string, txId: string, amount: number, vout: number): Promise<string> {
+  async send(addressTo: string, txId: string, amount: number, vout: number, btcFee: number): Promise<string> {
     return this.callNode(
       (c) =>
         c.call(
           NodeCommand.SEND,
           [
-            [{ [addressTo]: this.roundAmount(amount - this.btcUtxoFee) }],
+            [{ [addressTo]: this.roundAmount(amount - btcFee) }],
```

Comment:
Need to multiply btcFee with vByte count. Use e.g. https://bitcoinops.org/en/tools/calc-size/ (single input/output)